src/main/resources/nl/dtls/fairdatapoint/database/rdf/repository/common/findChildTitles.sparql

```diff
@@ -0,0 +1,6 @@
+prefix dct: <http://purl.org/dc/terms/>
+
+SELECT ?child ?title WHERE {
+	?parent ?relation ?child .
+	?child dct:title ?title .
```

Comment:
Is it always `dct:title` (not sure now if that can be somehow changed in shape/resource definition)?

Reply:
We could also consider a [property path](https://www.w3.org/TR/2013/REC-sparql11-query-20130321/#propertypaths): `?child dct:title|rdfs:label ?title .`, assuming the `rdfs:label` is always present (best practice) and might have a `sh:equals dct:title` [constraint](https://www.w3.org/TR/shacl/#EqualsConstraintComponent).

Reply:
As far as I remember, we decided a long time ago that `dct:title` should be present on all entities. It is used in many other places now, such as entity detail or search results. If we want to make this more flexible, we should change it in all places, and I think that Is out of the scope of this issue.